.git-authors

```diff
@@ -8,14 +8,15 @@ pairs:
   nw: Nathan Wang; nathan.wang2
   sc: Serena Cao; wcao0630
   mw: Matt Wills; mattwills8
+  ag: Arun Giridharan; linkin-park
   nc: Nelson Cothran; nelsonc19
 
-
 email:
   prefix: pair
   domain: gmail.com
 
 email_addresses:
    kj: karthijey@yahoo.com
    km: klausmana@outlook.com
-   mw: matt_wills8@hotmail.co.uk
\ No newline at end of file
+   mw: matt_wills8@hotmail.co.uk
+   ag: giridharan.arun123
```

Comment:
If you are not using gmail, you should be using your entire email here

Reply:
okay , will get it removed.